jest.config.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+export default {
+  displayName: 'datamanager',
+  preset: '../../jest.preset.js',
```

Comment:
path goes out of scope of the project, is that intentional?

Reply:
Yeah, in general the idea behind of NX is that libs and apps share the same scope, and all these configs are on NX root folder